docker-cloud/cloud-swarm/index.md

```diff
@@ -0,0 +1,160 @@
+---
+description: Work with swarms in Docker Cloud
+keywords: swarm mode, swarms, orchestration Cloud, fleet management
+title: Swarms in Docker Cloud
+---
+
+<b>Note</b>: All Swarm management features in Docker Cloud are free while in Beta.
```

Comment:
I've found it doesn't play well with the curly-brace style injection? And I was asked not to use HTML for centering.